test/end2end.py

```diff
@@ -204,3 +217,26 @@ def test_hanging_fatal(self):
 
         regex = re.compile("mympirun has been running for .* seconds without seeing any output.")
         self.assertTrue(len(regex.findall(out)) == 1, "Pattern '%s' found in: %s" % (regex.pattern, out))
+
+
+    def test_option_double(self):
+        """Test --double command line option"""
+        install_fake_mpirun('mpirun', self.tmpdir, txt=FAKE_MPIRUN_MACHINEFILE)
+        cmd = "%s %s --setmpi impirun --double hostname"
+        ec, out = run_simple(cmd % (sys.executable, self.mympiscript))
+        self.assertEqual(len(out.split('\n')), 4)
```

Comment:
please add a comment why `4` is the right answer here (`set_PBS_env` mimics `nodes=2:ppn=1` environment)